### PR TITLE
Add OrderPlace, LineItem, and Order classes

### PR DESCRIPTION
**Pull request by duartepinto**
Added the new classes:
* `VSDKOrderPlace`
* `VSDKLineItem`
* `VSDKOrder`
* `VSDKDiscount`

Removed deprecated classes `VSDKPurchase`, `VSDKRefund` and others that only existed because of these

Should be in accordance with https://docs.velocidi.com/collection/event-collection/web-and-e-commerce/#orders

### CODE REVIEW

File: VelocidiSDK/VelocidiSDK/Tracking/Model/VSDKLineItem.h [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+
+/**
+  An identifier for the product group
+ */
+@property (nullable) NSString *itemGroupId;
+
+/**
+  If the purchase relates to a subscription, this attribute captures the subscription duration, in days (Optional)
+ */
+@property int subscritpionDuration;
```
Comment (dimamo5) [inline VelocidiSDK/VelocidiSDK/Tracking/Model/VSDKLineItem.h]:
```suggestion
@property int subscriptionDuration;
```

File: VelocidiSDK/VelocidiSDK/Tracking/Model/VSDKDiscount.h [hunk 2]
```diff
@@ -0,0 +1,8 @@
+@import JSONModel;
+
+
+@interface VSDKDiscount : JSONModel
+
+@property float value;
```
Comment (dimamo5) [inline VelocidiSDK/VelocidiSDK/Tracking/Model/VSDKDiscount.h:6]: Missing `currency` field

File: VelocidiSDK/VelocidiSDK/Tracking/Model/VSDKOrder.h [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+@property float refund;
+
+/**
+  Payment type (e.g. credit card, debit card, Paypal)
+ */
+@property (nonnull) NSString *paymentMethod;
+
+@property (nonnull) NSString *shippingMethod;
+
+@property (nonnull) NSString *shippingCountry;
```
Comment (dimamo5) [inline VelocidiSDK/VelocidiSDK/Tracking/Model/VSDKOrder.h]: I think these 3 fields can be null.

**Review by dimamo5 — CHANGES_REQUESTED** [review]

File: VelocidiSDK/VelocidiSDK/Tracking/Model/VSDKOrder.h (hunk 3 above)
Comment (duartepinto) [reply VelocidiSDK/VelocidiSDK/Tracking/Model/VSDKOrder.h]: Change in 8e3decf

File: VelocidiSDK/VelocidiSDK/Tracking/Model/VSDKDiscount.h (hunk 2 above)
Comment (duartepinto) [reply VelocidiSDK/VelocidiSDK/Tracking/Model/VSDKDiscount.h:6]: Changed in 8e3decf

**Review by dimamo5 — APPROVED** [review]
👍

**Pull request merged**